ui/intermittent-failures/Layout.jsx

```diff
@@ -89,10 +89,7 @@ Layout.propTypes = {
   header: PropTypes.oneOfType([PropTypes.shape({}), PropTypes.bool]),
   table: PropTypes.oneOfType([PropTypes.shape({}), PropTypes.bool]),
   graphOneData: PropTypes.arrayOf(PropTypes.shape({})),
-  graphTwoData: PropTypes.arrayOf(
-    PropTypes.arrayOf(PropTypes.shape({})),
-    PropTypes.arrayOf(PropTypes.shape({})),
-  ),
+  graphTwoData: PropTypes.arrayOf(PropTypes.shape({})),
```

Comment:
For more context why this was changed, while working on the `prop-types` i noticed that the below console warning:

   ![image](https://user-images.githubusercontent.com/36057474/79585581-e65e2c00-80c7-11ea-9c15-6174f29d53d3.png)

So I did a little digging and discovered that the prop is an array of two objects, instead of an array of two arrays of objects, as shown in the react dev tool below.

   ![image](https://user-images.githubusercontent.com/36057474/79585673-0988db80-80c8-11ea-8d96-0490425085e5.png)

What I am not sure about is if I should use an array of two objects as in the picture above.
```jsx
  graphTwoData: PropTypes.arrayOf(
    PropTypes.shape({
      color: PropTypes.string,
      data: PropTypes.arrayOf(PropTypes.shape({})),
    }),
  ),
```
Tracing where the prop is been used, I discovered that the `Graph` component is using the same prop definition as above.
https://github.com/mozilla/treeherder/blob/a965e7b62fecdd4e624bc4451435c7b1f264c459/ui/intermittent-failures/Graph.jsx#L44-L49

Reply:
Yes, the propType should accurately represent whatever data structure you see - so in this case, an array of two objects.

Reply:
Okay, I will change to an array of two objects. Something like this:

```jsx
  graphTwoData: PropTypes.arrayOf(
    PropTypes.shape({ 
     data: PropTypes.arrayOf(PropTypes.shape({})), 
     color: PropTypes.string, 
   }), 
   PropTypes.shape({ 
     data: PropTypes.arrayOf(PropTypes.shape({})), 
     color: PropTypes.string, 
   }), 
  )
```